indico/modules/groups/templates/group_details.html

```diff
@@ -15,22 +15,24 @@
             {% trans %}Delete group{% endtrans %}
         </button>
     {% endif %}
-    <h3>{% trans %}Group members{% endtrans %}</h3>
-    <div id="group-members">
-        {% if group.is_local %}
-            {# Local groups usually don't have massive amounts of members so we load them immediately #}
-            {{ group_members(group) }}
-        {% else %}
```

Comment:
What about keeping the heading but showing a message here in `{% elif not group.supports_member_list %}` so it doesn't look like there's something missing on the page?

Something like "The data source of this group does not allow retrieving its member list."

Reply:
It sounds acceptable :)